lms/djangoapps/edxnotes/helpers.py

```diff
@@ -141,15 +137,13 @@ def preprocess_collection(user, course, collection):
     store = modulestore()
     filtered_collection = list()
     cache = {}
+    include_path_info = ('course_structure' not in settings.NOTES_DISABLED_TABS)
```

Comment:
if `course_structure ` is present in `settings.NOTES_DISABLED_TABS` then it means we don't need to calculate path info because `Location in Course` tab is disabled.